ios/Runner/Info.plist

```diff
@@ -25,7 +25,7 @@
 	<key>NSCameraUsageDescription</key>
 	<string>Uses camera to take pictures for profile.</string>
 	<key>NSFaceIDUsageDescription</key>
-	<string>FaceID can be used for local authorization</string>
```

Comment:
Do we need another description for TouchID? On devices that don't have FaceID?

Reply:
I can't find any reference to that so I guess FaceID is special since it uses the camera.